include/service/entities/site_service.js

```diff
@@ -598,13 +602,16 @@ module.exports = function SiteServiceModule(pb) {
      * @return {Object}
      */
     SiteService.getGlobalSiteContext = function() {
+        var defaultLocale = pb.Localization.getDefaultLocale();
+        var defaultSupportedLocales = {};
+        defaultSupportedLocales[defaultLocale] = true;
         return {
             displayName: pb.config.siteName,
             uid: pb.SiteService.GLOBAL_SITE,
             hostname: pb.config.multisite.enabled ? url.parse(pb.config.multisite.globalRoot).host : url.parse(pb.config.siteRoot).host,
             active: pb.config.multisite.enabled ? false : true,
-            defaultLocale: pb.Localization.getDefaultLocale(),
-            supportedLocales: util.arrayToObj(pb.Localization.getSupported(), function(a, i) { return a[i]; }, function(a, i) { return true; }),
+            defaultLocale: defaultLocale,
+            supportedLocales: defaultSupportedLocales,
```

Comment:
Rockin. That addresses my concern then. So then we're an object everywhere. I'll push that back momentarily